riscv/debug_module.cc

```diff
@@ -78,6 +68,14 @@ void debug_module_t::reset()
   abstractcs.progbufsize = progbufsize;
 
   abstractauto = {0};
+
+  sbcs = {0};
```

Comment:
Is system bus access a hard-coded feature? As well as the access types?

Reply:
Yes.

Reply:
Can we make it optional with a command line arg? S.t. SW can be tested against one that doesn't have system bus?

Reply:
Sure. Do you have suggestions for an option name? I'm having a hard time thinking of something concise but descriptive. Maybe --debug-sba? What do you want the default to be?

Currently in my testing I rely on OpenOCD to prefer the program buffer if it exists.

Reply:
I think --debug-sba=<N> , where N is the maximum size it supports and the default is 0? (I don't think it makes sense to support some size and not support something smaller than that).

Reply:
Done.